src/BizHawk.Emulation.Cores/Consoles/Nintendo/BSNES/BsnesCore.IVideoProvider.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System;
+using BizHawk.Emulation.Common;
+
+namespace BizHawk.Emulation.Cores.Nintendo.BSNES
+{
+	public partial class BsnesCore : IVideoProvider
+	{
+		// TODO: This should probably be different for PAL?
+		public int VirtualWidth => (int) Math.Ceiling((double) BufferHeight * 64 / 49);
+
+		public int VirtualHeight => BufferHeight;
+
+		public int BufferWidth { get; private set; } = 256;
```

Comment:
Is support for hires games a future endeavor?  That's fine with me so long as this doesn't absolutely crash out or anything on hires.

Reply:
This value is just a default. The actual value gets set every frame in `snes_video_refresh`.